rpc_jobs/rpc_octavia.yml

```diff
@@ -42,7 +44,8 @@
     # to Pike we will retire os-octavia and use the
     # upstream installer
     branches:
-      - "master"
+      - "master.*"
+      - "pike.*"
```

Comment:
Is there any protection in place to prevent this PM job from changing the artifacts it produces for the RC? If not, then this should be removed.